.travis.yml

```diff
@@ -14,3 +14,7 @@ script:
   - coverage run --omit=*/tests/* --source=hc manage.py test
 after_success: coveralls
 cache: pip
+
```

Comment:
This addition is irrelevant to this task. Each PR should complete one story at a time. (Single Responsibility Priciple'ish) ;)